api/nodes/views.py

```diff
@@ -672,9 +1004,32 @@ class NodeChildrenList(generics.ListCreateAPIView, NodeMixin, ODMFilterMixin):
 
     To create a child node of the current node, issue a POST request to this endpoint.  The `title` and `category`
     fields are mandatory. `category` must be one of the [permitted node categories](/v2/#osf-node-categories).  If the
-    node creation is successful the API will return a 201 response with the respresentation of the new node in the body.
+    node creation is successful the API will return a 201 response with the representation of the new node in the body.
     For the new node's canonical URL, see the `links.self` field of the response.
 
+
+    ###Create Multiple Child Nodes
```

Comment:
If the bulk extension is not going to be a part of the public API, do not include it in the documentation.